fillers/withdrawals/withdrawals.py

```diff
@@ -597,7 +599,7 @@ def test_withdrawals_zero_amount(_):
         amount=1,
     )
     block.withdrawals.append(withdrawal_2)
-    post[to_address(0x200)].balance = 1
+    post[to_address(0x200)].balance = ONE_GWEI
     yield BlockchainTest(pre=pre, post=post, blocks=[block])
```

Comment:
Could we add a new test case that uses the maximum value for the amount field in the withdrawal? I.e. use 2**64 -1 in amount. It should pass without issues.

Reply:
Done! I've added to the commit below